crates/common_enums/src/enums.rs

```diff
@@ -663,6 +715,11 @@ impl Currency {
             | Self::CLP
             | Self::DJF
             | Self::GNF
+            // ISO 4217 considers it zero decimal,
+            // Stripe considers it zero decimal since 2023-04-14,
+            // Braintree considers it zero decimal,
+            // Adyen however needs us to send two decimals to submit amounts in ISK
+            | Self::ISK
```

Comment:
@SanchithHegde Let's not support ISK for now, I will take it in separate PR, where decision will be made based on the connector also